packages/react-filerobot-image-editor/src/components/common/AnnotationOptions/index.jsx

```
@@ -54,17 +55,19 @@ const AnnotationOptions = ({
       },
       ...(!useCloudimage
         ? [
-            { titleKey: 'stroke', name: POPPABLE_OPTIONS.STROKE, Icon: Stroke },
+            ...(hideStrokeField ? [{ titleKey: 'stroke', name: POPPABLE_OPTIONS.STROKE, Icon: Stroke }] : []),
```

ESLint

[error] Replace `·?·[{·titleKey:·'stroke',·name:·POPPABLE_OPTIONS.STROKE,·Icon:·Stroke·}]` with `⏎··············?·[⏎··················{⏎····················titleKey:·'stroke',⏎····················name:·POPPABLE_OPTIONS.STROKE,⏎····················Icon:·Stroke,⏎··················},⏎················]⏎·············`

(prettier/prettier)